01-JS-DOM-APIs/js/index.js

```diff
@@ -0,0 +1,388 @@
+'use strict';
+
+/* The global variable Clicks advices how many times the user has pressed the joke button. Tt's used in EX 6, to
+ * control the Time  Out error. */
+var Clicks = 0;
```

Comment:
variables which are not intended to be used for Classes should start with lower case, please update this in all matching variables. Thanks!